app/src/main/java/com/infomaniak/mail/utils/ConfettiUtils.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Infomaniak Mail - Android
+ * Copyright (C) 2023 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.mail.utils
+
+import android.content.Context
+import android.view.ViewGroup
+import com.infomaniak.mail.MatomoMail.trackEasterEggEvent
+import com.infomaniak.mail.R
+import com.infomaniak.mail.confetti.CommonConfetti
+import com.infomaniak.mail.confetti.ConfettiSource
+import io.sentry.Sentry
+import io.sentry.SentryLevel
+
+object ConfettiUtils {
+
+    private const val EMISSION = 1_000_000.0f
+
+    fun triggerEasterEggConfetti(from: String, container: ViewGroup, context: Context): Unit = with(context) {
```

Comment:
We should rename this from parameter, as to something like "matomoFrom" to understand when we use this function that this should not be changed afterhand